Player/Player.gd

```diff
@@ -181,6 +184,20 @@ func _process_movement(delta: float) -> void:
 	vel.x = hvel.x
 	vel.z = hvel.z
 	
+	if(vel.y < 0):
+		fallHeight -= 0.1
```

Comment:
Countdowns like this should include the delta variable to make them framerate independent. Even though this variable is called fallHeight, it's really a timer, so it should be named to reflect that or should be changed to just measure the y distance between the zenith of the player's time in the air and the place where they land on the ground.
If it's changed to measure the distance, that would make it easier to know exactly how high the player can fall from without dying.